deployer/pom.xml

```diff
@@ -16,23 +16,118 @@
+            <artifactId>brooklyn-test-support</artifactId>
+            <version>${brooklyn.version}</version>
+            <scope>test</scope>
+        </dependency>
+
+        <dependency>
+            <groupId>org.apache.brooklyn</groupId>
+            <artifactId>brooklyn-core</artifactId>
+            <version>${brooklyn.version}</version>
+            <classifier>tests</classifier>
+            <scope>test</scope>
+        </dependency>
 
-  <groupId>eu.seaclouds-project</groupId>
-  <artifactId>deployer</artifactId>
-  <packaging>jar</packaging>
+        <dependency>
+            <groupId>org.apache.brooklyn</groupId>
+            <artifactId>brooklyn-launcher</artifactId>
+            <version>${brooklyn.version}</version>
+            <scope>test</scope>
+        </dependency>
 
-  <name>deployer</name>
-  <url>http://maven.apache.org</url>
+        <dependency>
+            <!-- this gives us flexible and easy-to-use logging; just edit logback-custom.xml! -->
+            <groupId>org.apache.brooklyn</groupId>
+            <artifactId>brooklyn-logback-xml</artifactId>
+            <version>${brooklyn.version}</version>
+            <!-- optional so that this project has logging; dependencies may redeclare or supply their own;
+                 provided so that it isn't put into the assembly (as it supplies its own explicit logback.xml);
+                 see Logging in the Brooklyn website/userguide for more info -->
+            <optional>true</optional>
+            <scope>provided</scope>
+        </dependency>
+    </dependencies>
 
-  <properties>
-    <project.build.sourceEncoding>UTF-8</project.build.sourceEncoding>
-  </properties>
+    <build>
+        <plugins>
+            <plugin>
+                <artifactId>maven-assembly-plugin</artifactId>
+                <configuration>
+                    <descriptors>
+                        <descriptor>src/main/assembly/assembly.xml</descriptor>
+                    </descriptors>
+                    <appendAssemblyId>false</appendAssemblyId>
+                </configuration>
+                <executions>
+                    <execution>
+                        <id>make-assembly</id>
+                        <!-- this is used for inheritance merges -->
+                        <phase>package</phase>
+                        <!-- bind to the packaging phase -->
+                        <goals>
+                            <goal>single</goal>
+                        </goals>
+                    </execution>
+                </executions>
+            </plugin>
 
+            <plugin>
+                <artifactId>maven-jar-plugin</artifactId>
+                <version>2.3.1</version>
+                <executions>
+                    <execution>
+                        <id>default-jar</id>
+                        <!-- put the default-jar in the none phase to skip it from being created -->
+                        <phase>none</phase>
```

Comment:
@andreaturli  I have deleted the deployer assembly and I have added all dependencies as resources to installer/pom.xml.
As I have mentioned above, I think that maintain this approach will be an error prone task, because we will have to maintain the integrity between the different dependencies versions if we modified any version.
If we add a new dependency, such as OpenShift client, we will add this dependency and its transitive dependencies as resource and to` BrooklyNode` classpath.

Moreover, I think that the number of dependencies that we need to add to the BrooklynNode classpath is too big.
Then, from my point of view I think it is easier the deployer assembly. However, it is just my opinion, the approach of this version works correctly so we could be to use it. Thoughts?